cdap-docs/hydrator-manual/source/creating-pipelines.rst

```diff
@@ -280,57 +279,52 @@ Real-time Pipelines
 
 Introduction
 ------------
-Real-time pipelines are designed to poll sources periodically to fetch data, perform any
-(optional) transformations, and then write to one or more real-time sinks. As they are
-intended to be run continuously, post-run actions are not applicable or available.
+Real-time pipelines are designed to generate micro batches of data at a regular interval, perform any
+(optional) transformations and analytics, and then write to one or more sinks. As they are
+intended to be run continuously, actions and post-run actions are not applicable or available.
 
 Types of Plugins
 ----------------
-Real-time pipelines, based on the ``cdap-etl-realtime`` application template, can include these plugins:
-
-- :ref:`Actions <cask-hydrator-action-plugins>`
+Real-time pipelines, based on the ``cdap-data-streams`` application template, can include these plugins:
 
-- :ref:`Real-time Sink Source Plugins <cask-hydrator-plugins-real-time-sources>`
+- :ref:`Streaming Source Plugins <cask-hydrator-plugins-real-time-sources>`
 
 - :ref:`Transformation Plugins <cask-hydrator-plugins-transformations>`
 
-- :ref:`Real-time Sink Plugins <cask-hydrator-plugins-real-time-sinks>`
+- :ref:`Analytics Plugins <cask-hydrator-plugins-analytics>`
+
+- :ref:`Batch Sink Plugins <cask-hydrator-plugins-batch-sinks>`
```

Comment:
Would be help to explain somewhere why "Batch" sink plugins can be used in "Real-time" pipelines? I'm of something along the lines of:

"Despite the name, batch sink plugins are not limited to just batch pipelines. The real-time pipeline artifact can use them as it is able to generate micro batches (*define a micro batch, or link to a definition, or link to an entry in the glossary*) that can then be written to a batch sink."